routes/index.js

```diff
@@ -1,7 +1,9 @@
 const router = require('express').Router();
 const api = require('./api');
 const routes = require('./routes');
+const { access } = require('./../auth');
 
+router.use((req, res, next) => access(req, res, next));
 router.use('/api', api);
 router.use('/', routes);
 
```

Comment:
https://nordicapis.com/introduction-to-api-versioning-best-practices/

Reply:
OK.